openquakeplatform/test/config.py.tmpl

```diff
@@ -0,0 +1,5 @@
+pla_basepath="http://localhost:8000"
+pla_user="admin"
+pla_passwd="admin"
+# path of debugger .xpi file extension
+pla_debugger=""
```

Comment:
Since this is not a `.py` file, if we want to have the test suite in the python package, we need to update also the `MANIFEST.in`